code/clue.js

```diff
@@ -51,9 +212,110 @@ const pickKiller = () => {
   document.getElementById(
     'killerName'
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerOccupation').innerHTML = `${mystery.killer.occupation}`
+  document.getElementById('killerDescription').innerHTML = `${mystery.killer.description}`
+  document.getElementById('killerImage').src = mystery.killer.image
 }
 
-// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
 
+
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponName').innerHTML = `${mystery.weapon.name}`;
+  document.getElementById('weaponWeight').innerHTML = `${mystery.weapon.weight}`
+  document.getElementById('weaponImage').src = mystery.weapon.image
+
+
+}
+
+const pickRoom = () => {
+  mystery.room = randomSelector(rooms)
+  document.getElementById('roomName').innerHTML = `${mystery.room}`
+}
+
+document.getElementById('killerCard').onclick = pickKiller
+document.getElementById('weaponCard').onclick = pickWeapon
+document.getElementById('roomCard').onclick = pickRoom
+
+// const guess1 = (name) => {
+//   guess = name
+//   document.getElementById('suspectImage').src = guess.image
+
+// }
+
+
+const reveal = () => {
+
+  if (mystery.killer === '' || mystery.weapon === '' || mystery.room === '') {
+    document.getElementById('mystery').innerHTML = 'Please select the mystery answer!'
+  } else if (guess === '') {
+    document.getElementById('mystery').innerHTML = 'Please make a guess!'
+  } else if (guess === mystery.killer) {
+    document.getElementById('mystery').innerHTML = `Correct! It was ${mystery.killer.firstName} ${mystery.killer.lastName} in the
+${mystery.room} with the ${mystery.weapon.name}.`
+  } else if (guess !== mystery.killer) {
+    document.getElementById('mystery').innerHTML = `Wrong! It was ${mystery.killer.firstName} ${mystery.killer.lastName} in the
+${mystery.room} with the ${mystery.weapon.name}.`
+  }
+}
+
+// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+document.getElementById('reveal').onclick = reveal
 // STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
 // 'The murder was committed by Jacob Green, in the living room with a rope.'
+
+let pickedSuspect
+
+const picked = (name, name2) => {
```

Comment:
The naming here could be confusing if you were using this function for the first time in a long time.
"Hmm, what is name?   What is name2?"

How could you make this clearer?